docroot/modules/custom/va_gov_facilities/src/EventSubscriber/FacilitiesSubscriber.php

```diff
@@ -69,9 +72,38 @@ public static function getSubscribedEvents(): array {
     return [
       EntityHookEvents::ENTITY_PRE_SAVE => 'entityPresave',
       EntityHookEvents::ENTITY_UPDATE => 'entityUpdate',
+      FieldHookEvents::WIDGET_COMPLETE_FORM_ALTER => 'widgetCompleteFormAlter',
     ];
   }
 
+  /**
+   * Creates custom array for render array.
+   *
+   * @param \Drupal\node\NodeInterface $node
+   *   Node that relates to the one whose field we want to render.
+   * @param string $related_field
+   *   Field that relates our node to another.
+   * @param string $field_to_render
+   *   Field we want to render.
+   *
+   * @return array
+   *   Render array
+   */
+  public static function createCustomArray(NodeInterface $node, $related_field, $field_to_render) {
```

Comment:
```suggestion
  public static function createRenderArrayFromFieldOnRefdEntity(NodeInterface $node, $related_field, $field_to_render) {
```
Specificity improves maintainability for our future selves.